projects/sandbox/pyproject.toml

```diff
@@ -100,20 +100,20 @@ outdir = "${base.basedir}/training"
 sample_rate = "${base.sample_rate}"
 kernel_length = 2
 fduration = "${base.fduration}"
-
+snr_thresh = "${base.hopeless_snr_thresh}$"
 highpass = "${base.minimum_frequency}"
 trigger_distance = -0.5
-waveform_prob = 0.5
-glitch_prob = 0.25
-glitch_downweight = 0.5
+waveform_prob = 0.275589
+glitch_prob = 0.281727
+glitch_downweight = 0.755453 
 
 # optimization parameters
 batch_size = 512
 batches_per_epoch = 200
 max_epochs = 200
-max_lr = 4e-3
-lr_ramp_epochs = 75 
-weight_decay = 1e-5
+max_lr = 0.21355 
+lr_ramp_epochs = 83 
+weight_decay = 6.28608e-06
```

Comment:
These params should be set back to their defaults.

Reply:
Might be a good time to update to the best params from the hp search. Can you share those here?

Reply:
Good idea:
```
waveform_prob: 0.470644, 
glitch_prob: 0.420902, 
max_lr: 0.000181352, 
lr_ramp_epochs: 92, 
weight_decay: 2.81536e-06,
glitch_downweight: 0.650283
```